src/microbe_stage/Biome.cs

```diff
@@ -22,6 +22,11 @@ public class Biome : IRegistryType
     /// </summary>
     public string Icon;
 
+    /// <summary>
+    ///   Depth of the patch below sea level in meters
+    /// </summary>
+    public int[] Depth;
```

Comment:
Is it possible to enforce with the type that this would always have 2 items, like does `int[2]`work?
Otherwise a check should be added to `Check` to make sure the data is valid.

Reply:
Array size cannot be specified in a variable declaration (try initializing with a 'new' expression) [Thrive]csharp(CS0270)

Reply:
then add the check to `Check`